docs_user/modules/proc_adopting-the-orchestration-service.adoc

```diff
@@ -2,64 +2,57 @@
 
 = Adopting the {orchestration}
 
-Adopting the {orchestration_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where {orchestration}
-is supposed to be disabled, should be patched to start the service with the
-configuration parameters provided by the source environment.
+To adopt the {orchestration_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR), where the {orchestration}
+is disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
 
-After the adoption process has been completed, a user can expect that they
-will then have CRs for `Heat`, `HeatAPI`, `HeatEngine` and `HeatCFNAPI`.
-Additionally, a user should have endpoints created within {identity_service_first_ref} to facilitate
-the above mentioned servies.
-
-This guide also assumes that:
-
-. A {OpenStackPreviousInstaller} environment (the source Cloud) is running on one side;
-. A {rhocp_long} environment is running on the other side.
+After you complete the adoption process, you have CRs for `Heat`, `HeatAPI`, `HeatEngine`, and `HeatCFNAPI`, and endpoints within the {identity_service_first_ref} to facilitate these services.
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably, MariaDB and {identity_service}
-should be already adopted.
-* In addition, if your existing {orchestration} stacks contain resources from other services
-such as {networking_first_ref}, {compute_service_first_ref}, {object_storage_first_ref}, etc. Those services should be adopted first before
-trying to adopt {orchestration}.
+* The source {OpenStackPreviousInstaller} environment is running.
+* The target {rhocp_long} environment is running.
+* You adopted MariaDB and the {identity_service}.
+* If your existing {orchestration} stacks contain resources from other services such as {networking_first_ref}, {compute_service_first_ref}, {object_storage_first_ref}, and so on, adopt those sevices before adopting the {orchestration}.
 
 .Procedure
 ifeval::["{build}" != "downstream"]
 As already done for https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md[Keystone], the Heat Adoption follows a similar pattern.
```

Comment:
Maybe: "The Heat Adoption is a similar workflow to https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md[Keystone]."